modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/ObjectField/ObjectField.js

```diff
@@ -95,6 +95,9 @@ const ObjectField = ({
 				else if (relationshipType) {
 					return false;
 				}
+				else if (system) {
```

Comment:
@danvb112 I think you could just add the check for the `system` value in the `else if` above :) 
```
else if(relationshipType || system) {
    return false;
}
```

Reply:
Thank you @naraandrade ! I hadn't noticed that! :)